migrations/20181025093830_add_unique_constraint_to_date_field_rhpasody_table_schema.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+exports.up = function (knex) {
+    return knex.schema
+        .table('rhapsody', function (table) {
+            table.date('date').unique().alter();
+        });
+};
+
+exports.down = function (knex) {
+    return knex.schema
+        .table('rhapsody', function (table) {
+            table.dropColumn('date');
```

Comment:
the `.down` portion is supposed to return back the table to how it was before, i.e it is expected to remove the `unique` property of that column, but what has been done here is to drop the column entirely - which is not what should be done

Reply:
found this, don't know if thats how to drop the unique constraint on the column, worth checking out
https://github.com/tgriesser/knex/issues/292